src/main/proto/ga4gh/assay_metadata.proto

```diff
@@ -0,0 +1,96 @@
+syntax = "proto3";
+
+package ga4gh;
+
+import "google/protobuf/struct.proto";
+
+
+// An experimental preparation of a sample.
+message Experiment {
+  // The experiment UUID. This is globally unique.
+  string id = 1;
+
+  // The name of the experiment.
+  string name = 2;
+
+  // A description of the experiment.
+  string description = 3;
+
+  // The time at which this message was created.
+  // Format: ISO 8601, YYYY-MM-DDTHH:MM:SS.SSS (e.g. 2015-02-10T00:03:42.123Z)
+  string message_create_time = 4;
```

Comment:
The time of time has not be changed yet in the other records.    

For now, lets make the name consistent the other records (created and updated), but leave the time ISO.  We can then take on global conversion to time ISO.

message_create_time is a confusing name anyway.  It should be abstracted away from protobuf.